src/constants/sinsemilla.rs

```diff
@@ -37,6 +37,18 @@ pub const Q_NOTE_COMMITMENT_M_GENERATOR: ([u8; 32], [u8; 32]) = (
     ],
 );
 
+/// Generator used in SinsemillaHashToPoint for ZSA note commitment
+pub const Q_NOTE_ZSA_COMMITMENT_M_GENERATOR: ([u8; 32], [u8; 32]) = (
+    [
+        207, 235, 191, 45, 66, 225, 8, 126, 199, 188, 39, 26, 115, 106, 18, 2, 191, 173, 75, 9, 65,
+        225, 175, 193, 224, 202, 228, 177, 3, 75, 228, 1,
+    ],
+    [
+        220, 251, 80, 86, 182, 182, 99, 67, 254, 97, 241, 22, 79, 111, 161, 176, 79, 97, 208, 98,
+        116, 57, 110, 196, 25, 73, 239, 31, 196, 97, 19, 30,
+    ],
+);
+
```

Comment:
How does this got generated?

Reply:
The test `q_note_zsa_commitment_m` checks that the values are correct.